README.md

```diff
@@ -1 +1,121 @@
 # py-google-auth
+
+## Introduction
+An api to log into a user's Google account using their email and password.
+
+## Set up for testing
+
+* Create Virtual Environment  
+(_Note: Make sure you have python [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/) and [virtualenvwrapper](http://docs.python-guide.org/en/latest/dev/virtualenvs/) installed_)        
+
+        mkvirtualenv -p `which python3.4` py-google-auth
+
+* Clone the repository        
+
+        git clone https://github.com/HashGrowth/py-google-auth.git py-google-auth
+
+* Install requirements        
+
+        pip install -r requirements.txt
+
+* Set an environment variable for logs, run this on terminal        
+
+        "export PY_GOOGLE_AUTH_LOG_PATH=/path/to/logs" >> ~/.bashrc
```

Comment:
Correct it else it won't work(add echo):

    echo "export PY_GOOGLE_AUTH_LOG_PATH=/path/to/logs" >> ~/.bashrc